native/src/main/java/io/ballerina/lib/ibm.ibmmq/CommonUtils.java

```diff
@@ -18,23 +18,180 @@
 
 package io.ballerina.lib.ibm.ibmmq;
 
+import com.ibm.mq.MQException;
+import com.ibm.mq.MQMessage;
+import com.ibm.mq.MQPropertyDescriptor;
+import io.ballerina.runtime.api.PredefinedTypes;
 import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.creators.TypeCreator;
+import io.ballerina.runtime.api.creators.ValueCreator;
 import io.ballerina.runtime.api.utils.StringUtils;
 import io.ballerina.runtime.api.values.BError;
 import io.ballerina.runtime.api.values.BMap;
 import io.ballerina.runtime.api.values.BString;
 
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Enumeration;
 import java.util.Optional;
 
+import static io.ballerina.lib.ibm.ibmmq.Constants.IBMMQ_ERROR;
+import static io.ballerina.lib.ibm.ibmmq.ModuleUtils.getModule;
+
 /**
  * {@code CommonUtils} contains the common utility functions for the Ballerina IBM MQ connector.
  */
 public class CommonUtils {
 
+    public static final String BTOPIC = "TOPIC";
+    public static final String TOPIC_OBJECT = "TOPIC_OBJECT";
```

Comment:
When naming the key for the native data I think following approach will give a more clear idea.
```suggestion
    public static final String NATIVE_TOPIC = "topic";
```